app/src/main/java/org/mozilla/focus/home/HomeTilesManager.kt

```diff
@@ -93,10 +93,10 @@ class BundledTilesManager private constructor(context: Context) {
                         .putStringSet(BUNDLED_SITES_ID_BLACKLIST, newBlacklist)
                         .apply()
                 bundledTilesCache.remove(pair.key)
-                return true
+                return pair.value.id
             }
         }
-        return false
+        return null
```

Comment:
I still don't love passing null around as a type, but I guess this is a not-too-messy way to get the tile id on removal, and other Java `.remove` functions do it too.